libs/dh/market-participant/feature-edit-actor/src/lib/market-participant-edit-actor/dh-market-participant-edit-actor.component.ts

```diff
@@ -95,8 +103,41 @@ export class DhMarketParticipantEditActorComponent {
     this.backToOverview();
   };
 
-  readonly onSaved = () => {
-    this.store.save(this.backToOverview);
+  readonly onSaved = (initialStatus: ActorStatus, changes: ActorChanges) => {
+    if (
+      (changes.status === ActorStatus.Inactive &&
+        initialStatus !== ActorStatus.Inactive) ||
+      (changes.status == ActorStatus.Passive &&
+        initialStatus != ActorStatus.Passive)
+    ) {
+      this.toast.open({
+        actionLabel: this.translocoService.translateObject(
+          'marketParticipant.actor.create.masterData.statusChangeWarningActionLabel'
+        ),
+        type: 'warning',
+        duration: 60000,
+        message: this.translocoService.translateObject(
+          'marketParticipant.actor.create.masterData.statusChangeWarning',
+          {
+            status: this.translocoService.translateObject(
+              'marketParticipant.actor.create.masterData.statuses.' +
+                changes.status
+            ),
+            initialStatus: this.translocoService.translateObject(
+              'marketParticipant.actor.create.masterData.statuses.' +
+                initialStatus
+            ),
+          }
+        ),
+        action: (ref: WattToastRef) => {
+          this.store.save(this.backToOverview);
+          ref.dismiss();
+        },
+      });
```

Comment:
suggestion: it works fine with a toast but the [modal](https://main--61765fc47451ff003afe62ff.chromatic.com/?path=/story/components-modal--small) is probably a better option here. Also because the modal wont auto close after some time. The `small` size should be fine.